api4/config.go

```diff
@@ -320,6 +323,16 @@ func makeFilterConfigByPermission(accessType filterType) func(c *Context, struct
 			if tagValue == model.ConfigAccessTagCloudRestrictable {
 				continue
 			}
+			if tagValue == AnyTagValue {
+				checkPermissions := model.SysconsoleReadPermissions
+				if accessType == FilterTypeWrite {
+					checkPermissions = model.SysconsoleWritePermissions
+				}
```

Comment:
I think this is not needed. A system admin is given feature flags in the config as long as they have access to the system console. If they have access to the system console, they must at the very least have read permissions. If they have write, they will always have READ+WRITE. So we can just check for sys console read permissions and not worry about the write.